src/table/table.scss

```diff
@@ -180,6 +186,24 @@
       }
     }
 
+    &.iui-expanded-content {
```

Comment:
Can we have this in HTML as an example?

Reply:
These classes come from react-transition-group, so I think not